### ISSUE TITLE
Feature request: More silent reading of EDF files

### ISSUE THREAD

**Issue by alexholcombe** [alexander-pastukhov/eyelinkReader#5]
Would be good to have a way to suppress the messages that for me always seem to occur when reading an EDF file, such as:

```
loadEvents = 1
setTrial End:  TRIAL_RESULT
loadEvents = 0
```
I confess I don't know exactly what these mean but they seem to reflect expected events, so it confuses users to see them printed out, especially in the context of code reading in a lot of EDFs, where these messages could drown out more informativ messages.

I don't understand the Eyelinks EDFAPI, but I'm wondering whether a silent=TRUE parameter could be optionally passed to e.g. extract_saccades, just as it [is passed ](https://github.com/alexander-pastukhov/eyelinkReader/blob/c948c9f673147956da177d90d606c110b01ebf4c/R/extract_display_coords.R)to extract_display_coords?

Maybe I could work on this if so. Thank you again for the package!

**Comment by alexander-pastukhov** [comment]
@alexholcombe I'll take a look. This is not my code but rather EDF API messages. I will try to figure out whether it is possible to suppress them. Just to be sure: This happens when you call read_edf(), correct?

**Comment by alexholcombe** [comment]
@alexander-pastukhov  yes, but it's only a minor thing; there are probably higher priorities for the package, but point me to the associated call if you find it; I don't understand how the C bindings work.

**Comment by alexander-pastukhov** [comment]
@alexholcombe I've reached out to SR-Research and apparently these are debugging messages accidentally left over. They promised to turn them off in the upcoming release of the DevKit. I'll close the issue once it is no longer an issue (just to remind myself not to forget to check it).
